lib/apress/selenium_eti/spec/company_site/eti/product_price_spec.rb

```diff
@@ -39,88 +42,122 @@
-          expect(@cs_eti_page.price_value).to include @price_from_to[:from].to_s, @price_from_to[:to].to_s
+          expect(@cs_eti_table_products.price(@product)).to include(price[:price].to_s, price[:price_max].to_s)
         end
       end
     end
 
     context 'когда цена со скидкой', story: 'когда цена со скидкой' do
-      before do
-        @discount_price = {previous: Faker::Number.number(digits: 3), discount: Faker::Number.number(digits: 2)}
-        @cs_eti_page.set_discount_price(@discount_price)
-        @cs_eti_page.wait_saving
-        @cs_eti_page.search_product(@name)
+      let(:price) do
+        {
+          type: :discount,
+          price: Faker::Number.number(digits: 3),
+          new_price: Faker::Number.number(digits: 2),
+          expires_at: Time.now.strftime('%d.%m.%Y')
+        }
       end
 
       it 'введеная цена отображается' do
-        expect(@cs_eti_page.discount_price_value).to include @discount_price[:discount].to_s
-        expect(@cs_eti_page.previous_price_value).to include @discount_price[:previous].to_s
+        expect(@cs_eti_table_products.price(@product)).to include(price[:price].to_s, price[:new_price].to_s)
+      end
+    end
+  end
+
+  describe 'Установка оптовых цен', feature: 'company_site/eti/product_price_spec: Установка оптовых цен' do
+    before do
+      @name = Faker::Number.number(digits: 5).to_s
+      @cs_eti_table_products.add_product(name: @name, wholesale_price: wholesale_price)
+      @cs_eti_header.search_product(@name, exact: true)
+      @product = @cs_eti_table_products.product(name: @name)
+    end
+
+    context 'когда точная оптовая цена', story: 'когда точная оптовая цена' do
+      let(:wholesale_price) do
+        {
+          price: Faker::Number.number(digits: 2),
+          min_qty: Faker::Number.non_zero_digit
+        }
+      end
+
+      it 'введеная цена отображается' do
+        if @cs_eti_table.project_pulscen?
+          expect(@cs_eti_table_products.wholesale_price(@product))
+          .to eq("#{wholesale_price[:price]} руб. /шт. от #{wholesale_price[:min_qty]} шт.")
+        elsif @cs_eti_table.project_blizko?
+          expect(@cs_eti_table_products.wholesale_price(@product))
+          .to eq("#{wholesale_price[:price]} ₽ /шт. от #{wholesale_price[:min_qty]} шт.")
+        else
+          raise ArgumentError, 'locator not found'
+        end
       end
     end
 
-    context 'когда цена оптовая', story: 'когда цена оптовая' do
-      before do
-        @price = {wholesale_price: Faker::Number.number(digits: 2), wholesale_number: Faker::Number.number(digits: 1)}
-        @cs_eti_page.set_wholesale_price(@price)
-        @cs_eti_page.wait_saving
-        @cs_eti_page.search_product(@name)
+    context 'когда оптовая цена "от"', story: 'когда оптовая цена "от"' do
+      let(:wholesale_price) do
+        {
+          price: Faker::Number.number(digits: 2),
+          min_qty: Faker::Number.non_zero_digit,
+          not_exact: true
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.